ui/components/multichain/account-list-menu/account-list-menu.tsx

```diff
@@ -282,7 +288,10 @@ export const AccountListMenu = ({
       history.push(CONFIRMATION_V_NEXT_ROUTE);
     }
 
-    await bitcoinWalletSnapClient.createAccount(network);
+    await bitcoinWalletSnapClient.createAccount(
+      network,
+      primaryKeyring.metadata.id,
+    );
```

Comment:
Dropping entropySource support for Bitcoin for now.

```suggestion
    // TODO: Forward the `primaryKeyring.metadata.id` to Bitcoin once supported.
    await bitcoinWalletSnapClient.createAccount(network);
```